Export issue (Module not found: Error: Can't resolve '@ledgerhq/wallet-api-client')

I try to implement wallet-api into my React application and going through the example
https://wallet.api.live.ledger.com/react/configuration

But I got this error - Module not found: Error: Can't resolve '@ledgerhq/wallet-api-client'

I tried to downgrade to the previous version of 'wallet-api-client' ("@ledgerhq/wallet-api-client": "^0.15.5") to check the path of imported Transport and WindowMessageTransport and I found that 'Transport' exported from - @ledgerhq/wallet-api-client/node_modules/@ledgerhq/wallet-api-core/lib/transports/types.d.ts
And the 'WindowMessageTransport' from -
@ledgerhq/wallet-api-client/node_modules/@ledgerhq/wallet-api-core/lib/transports/WindowMessageTransport.d.ts

But in version 1.0.0 path is different and even if I put the entire way to the 'Transport' and 'WindowMessageTransport' (like '@ledgerhq/wallet-api-client/lib/core/src/transports/WindowMessageTransport') still got an import error.

The same issue for the 'WalletAPIProvider' from 'wallet-api-client-react'.

Could you please check the exports.

Thanks

--- a/.changeset/rare-roses-cross.md
+++ b/.changeset/rare-roses-cross.md
@@ -1,0 +1,10 @@
+---
+"@ledgerhq/wallet-api-manifest-validator": patch
+"@ledgerhq/wallet-api-client-react": patch
+"@ledgerhq/wallet-api-simulator": patch
+"@ledgerhq/wallet-api-client": patch
+"@ledgerhq/wallet-api-server": patch
+"@ledgerhq/wallet-api-core": patch
+---
+
+fix: missing baseUrl for prod tsconfig

--- a/packages/client-react/prod-esm.tsconfig.json
+++ b/packages/client-react/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/client-react/prod.tsconfig.json
+++ b/packages/client-react/prod.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "commonjs",
     "outDir": "./lib"
   },

--- a/packages/client/prod-esm.tsconfig.json
+++ b/packages/client/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/client/prod.tsconfig.json
+++ b/packages/client/prod.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "commonjs",
     "outDir": "./lib"
   },

--- a/packages/core/prod-esm.tsconfig.json
+++ b/packages/core/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/core/prod.tsconfig.json
+++ b/packages/core/prod.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "commonjs",
     "outDir": "./lib"
   },

--- a/packages/manifest-validator/prod-esm.tsconfig.json
+++ b/packages/manifest-validator/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/manifest-validator/prod.tsconfig.json
+++ b/packages/manifest-validator/prod.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "commonjs",
     "outDir": "./lib"
   },

--- a/packages/server/prod-esm.tsconfig.json
+++ b/packages/server/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/server/prod.tsconfig.json
+++ b/packages/server/prod.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "commonjs",
     "outDir": "./lib"
   },

--- a/packages/simulator/prod-esm.tsconfig.json
+++ b/packages/simulator/prod-esm.tsconfig.json
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
     "module": "es2022",
     "outDir": "./lib-es"
   },

--- a/packages/simulator/prod.tsconfig.json
+++ b/packages/simulator/prod.tsconfig.json
@@ -1,4 +1,9 @@
 {
   "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "module": "commonjs",
+    "outDir": "./lib"
+  },
   "include": ["src/**/*"]
 }

